hw/top_darjeeling/dv/chip_sim_cfg.hjson

```diff
@@ -1673,15 +1674,6 @@
 
   // List of regressions.
   regressions: [
-    { name: babo
```

Comment:
looks like this edit should be rebased into the first commit...

Reply:
rebased